dolphinscheduler-ui/src/views/projects/task/components/node/format-data.ts

```diff
@@ -315,6 +315,7 @@ export function formatParams(data: INodeData): {
   if (data.taskType === 'ZEPPELIN') {
     taskParams.noteId = data.zeppelinNoteId
     taskParams.paragraphId = data.zeppelinParagraphId
+    taskParams.parameters = data.parameters
```

Comment:
Have you checked the tsc locally？

Reply:
@Amy0104 I've tested locally and it looks fine. Not sure what `tsc` mean?

Reply:
I have checked the code, and this is ok for the tsc. Sorry, I have confused you.
<img width="649" alt="image" src="https://user-images.githubusercontent.com/97265214/171085817-6e1d3b6c-dc9b-421e-8f86-050326b675ff.png">

Reply:
No worries, thx for help me with the review! : )